lib/Search/UI/SearchResultsController.php

```diff
@@ -202,7 +202,7 @@ protected function getListViewHeaderLabel(SugarBean $bean, $fieldValue, $fieldDe
                 $fieldValue['label'],
                 null,
                 LangText::USING_ALL_STRINGS,
-                true,
+                false,
                 true,
```

Comment:
@Dillon-Brown was looking into the LangText code. if we set log as `false`, throw will never happen. That said maybe best to also set `throw` as false here, to avoid having people confused when looking at this code in the future.

Reply:
Ah, nice spot. I actually intended to set throw as false and keep logging. Pushed.

Reply:
yup, I though of that also, but if we set the log to true, the text will be null (or similar). However, when you set the log to false, it sets the text to be the same as the key.

Should it be empty or have the key?

```php
        if (!$text) {
            if ($this->log) {
                ErrorMessage::handler('A language key does not found: [' . $this->key . ']', self::LOG_LEVEL, $this->throw);
            } else {
                $text = $this->key;
            }
        }
```

Reply:
Should be empty, wouldn't want to display invalid keys as text, better to just log it and move on imo.

Reply:
agreed!